src/pages/en/tutorial/1-setup/1.mdx

```diff
@@ -26,15 +26,22 @@ You will use a **command line (terminal)** to create your Astro project and to r
 
 You can access the command line through a local terminal program for your operating system. Common applications include **Terminal** (MacOS/Linux), **Command Prompt** (Windows), and **Termux** (Android). One of these will probably already be on your machine. 
 
-For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed. To check to see whether you already have a compatible version (`v16.12.0` or higher) installed: 
+### Node.js
 
-1. Launch your terminal and type the following into the command line:
+For Astro to run on your system, you also need to have [**Node.js**](https://nodejs.org/en/) of version `v16.12.0` or later.
+First, check whether you already have Node.js installed and check the installed version. To do this, run the following command in your terminal:
```

Comment:
```suggestion

To check to see whether you already have a compatible version (`v16.12.0` or higher) installed, run the following command in your terminal:
```
Looking at the preview, these lines run together. So, I think an extra line space is needed here.

And, I'm not sold on the proposed change. It seems overly wordy (check and check; then refering back to it again with "this") when we currently say "check that you have a compatible version installed" which covers both cases directly.

Reply:
> It seems overly wordy

[npm docs](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#checking-your-version-of-npm-and-nodejs) use almost the same wording.

> "check that you have a compatible version installed" which covers both cases directly.

No, it doesn't. Again, the current version doesn't take into acount the negative flow, when the reader doesn't have Node at all.

Let's read the current version: `To check to see whether you already have a compatible version ... installed` What outcomes are expected with this phrase?
The reader has either a compatible version installed, or an incompatible one.
But there is a third possible outcome, which is not covered by the current version: when the reader **gets an error** after performing `node -v` because they have no Node installed in their system. 

Yes, it is a very unlikely scenario considering the target audience experience and skills, but it is still a possible outcome.

The first Technical Writer's dogma is: "Don't expect your readers to be as smart as you are".

Reply:
> "To check to see whether you already have a compatible version (`v16.12.0` or higher) installed" 
> What outcomes are expected with this phrase?

The possible outcomes are:
- You already have a compatible version installed
- You don't have a compatible version installed
    - _Within_ this second outcome, you could either have no version installed at all, or you could have an incompatible version.

I agree that Sarah's text here is less clunky and achieves the same result, especially given your revisions below to explicitly anticipate the error case in the next section.

Reply:
Yes, It kinda makes sense. I just wanted to be a bit more clear. If you feel like the readers won't have troubles understanding this, I'm totally OK with that.
Anyway, we can always handle this based on feedback